src/ScriptCs.Core/ReplCommands/ScriptPacksCommand.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using ScriptCs.Contracts;
+using ScriptCs.Extensions;
+
+namespace ScriptCs.ReplCommands
+{
+    public class ScriptPacksCommand : IReplCommand
+    {
+        private readonly IConsole _console;
+
+        public ScriptPacksCommand(IConsole console)
+        {
+            _console = console;
+        }
+
+        public string Description
+        {
+            get { return "Displays information about script packs available in the REPL session"; }
+        }
+
+        public string CommandName
+        {
+            get { return "scriptpacks"; }
+        }
+
+        public object Execute(IRepl repl, object[] args)
+        {
+            var packContexts = repl.ScriptPackSession.Contexts;
+            var originalColor = _console.ForegroundColor;
+            _console.ForegroundColor = ConsoleColor.Yellow;
+
+            if (packContexts.IsNullOrEmpty())
+            {
+                _console.WriteLine("There are no script packs available in this REPL session");
+                _console.ForegroundColor = originalColor;
+                return null;
+            }
+
+            var importedNamespaces = repl.ScriptPackSession.Namespaces.IsNullOrEmpty() ? repl.Namespaces.ToArray() : repl.Namespaces.Union(repl.ScriptPackSession.Namespaces).ToArray();
+
+            foreach (var packContext in packContexts)
+            {
+                var contextType = packContext.GetType();
+                
+                _console.WriteLine(contextType.ToString());
+                _console.ForegroundColor = originalColor;
```

Comment:
you are right, i think i messed it up at some point (look at the screenshot, it was ok)